content-repo/extra-docs/releases/20.7.0.md

```diff
@@ -0,0 +1,516 @@
+  - Syncs data with Cortex XDR.
+  - Notifies management about a compromised host.
+  - Escalates the incident in case of lateral movement alert detection.
+
+The playbook is used as a sub- playbook in 'Cortex XDR Incident Handling - v2'
+
+##### New: Cortex XDR Alerts Handling    
+This playbook is used to loop over every alert in a Cortex XDR incident.
+Supported alert categories:
+- Malware
+- Port Scan
+
+##### New: Cortex XDR Incident Handling v2    
+This playbook is triggered by fetching a Palo Alto Networks Cortex XDR
+incident.
+
+---
+
+### Palo Alto Networks PAN-OS EDL Service Pack v1.0.3
+#### Playbooks
+##### PAN-OS EDL Service Configuration  
+Added a conditional task to validate if a EDL security rule already exists with the same name.
+
+---
+
+### Phishing Pack v1.7.0
+#### Playbooks
+##### Process Email - Generic  
+The task that updates the email headers in the layout will no longer continue on errors.
+
+##### Process Email - Core  
+The task that updates the email headers in the layout will no longer continue on errors.
+
+---
+
+### Proofpoint Feed Pack v1.0.1
+#### Integrations
+##### Proofpoint Feed  
+Added the *Tags* parameter.
+
+---
+
+### Rasterize Pack v1.0.3
+#### Integrations
+##### Rasterize  
+- Fixed an issue where the ***rasterize-image*** command returned an image instead of a pdf file.
+- Added support for additional languages.
+
+---
+
+### Recorded Future Feed Pack v1.0.2
+#### Integrations
+##### Recorded Future RiskList Feed  
+Added the *Tags* parameter.
+
+---
+
+### Remedy On-Demand Pack v1.0.1
+#### Integrations
+##### Remedy On-Demand  
+Fixed an issue where the ***remedy-incident-update*** and the ***remedy-get-incident*** commands required the request ID instead of the entry ID.
+
+---
+
+### RiskSense Pack v1.0.1
+#### Integrations
+##### RiskSense  
+added new command risksense-apply-tag that is a part of playbook for applying tag.
+
+#### Playbooks
+##### CVE Exposure - RiskSense  
+Block IPs and apply the tag to assets that are vulnerable to specified CVE.
+
+#### Scripts
+##### DisplayCVEChartScript  
+A script that displays a bar chart based on CVEs count and trending CVEs count with the different colors.
+
+---
+
+### ServiceNow Pack v1.1.5
+#### Integrations
+##### ServiceNow v2  
+Fixed the test button to work with debug mode.
+
+---
+
+### Slack Pack v1.3.1
+#### Integrations
+##### SlackV2  
+Increased integration context reliability by using versions (supported in Cortex XSOAR v6.0 and later).
+
+---
+
+### Symantec Management Center Pack v1.0.1
+#### Integrations
+##### Symantec Management Center
+Added support for content type LOCAL_CATEGORY_DB.
+
+---
+
+### TAXII Feed Pack v1.0.1
+#### Integrations
+##### TAXII Feed  
+Added the *Tags* parameter.
+
+---
+
+### ThreatConnect Pack v2.0.1
+#### Integrations
+##### ThreatConnect  
+Deprecated. Use the ThreatConnect v2 integration instead
+
+##### ThreatConnect v2  
```

Comment:
```suggestion
##### New: ThreatConnect v2  
```
@yaakovi fyi. Also can we add a line or two specifying what is new in v2 compared to v1.